dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/model/TaskNode.java

```diff
@@ -393,7 +394,9 @@ public void setPreTaskNodeList(List<PreviousTaskNode> preTaskNodeList) {
     }
 
     public String getTaskParams() {
-        Map<String, Object> taskParams = JSONUtils.toMap(this.params, String.class, Object.class);
+//        Map<String, Object> taskParams = JSONUtils.toMap(this.params, String.class, Object.class);
```

Comment:
Please remove the unused code

Reply:
Oh I forgot to remove this one. Thank you for pointing it out.

Reply:
@EricPyZhou Please fix the ci error.